implementation/domain/api/jbb-system-api/src/main/java/org/jbb/system/api/model/session/UserSession.java

```diff
@@ -0,0 +1,25 @@
+package org.jbb.system.api.model.session;
+
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+
+public interface UserSession {
```

Comment:
Ok, we'll schedule refactor in nearest future. We need to get rid of those naming convention